examples/gatekeeper-imperative/README.md

```diff
@@ -15,21 +23,15 @@ We have a `ConfigMap` in `config-map.yaml` that violates the policy.
 
 ### Function invocation
 
-Get the package:
-
-```shell
-$ kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/gatekeeper/imperative
-```
-
 Run the function with `--results-dir` flag:
 
 ```shell
-$ kpt fn eval --image=gcr.io/kpt-fn/gatekeeper:unstable --results-dir=results
+$ kpt fn eval gatekeeper-imperative --image=gcr.io/kpt-fn/gatekeeper:unstable --results-dir=/tmp
```

Comment:
not a big deal. `=` is not needed here. just `--results-dir /tmp` is pleasing to the eyes.